ui/apps/platform/src/css/trumps.css

```diff
@@ -4,6 +4,11 @@
    We should have a goal of eventually being able to delete this file.
  */
 
+body {
+    /* Overrides the tailwind default font-family that clashes with the upgrade to PF 5 */
+    font-family: var(--pf-v5-global--FontFamily--text);
+}
```

Comment:
Apparently global PatternFly `font-family` style rule changed so deletion of `font-family` property from the following style rule in #5977